### DISCUSSION TITLE
请问怎么调节参考文献里的字体大小和行间距？

### DISCUSSION THREAD

**Discussion by sastars9** [nju-lug/NJUThesis#193] [Q&A] [answered]
不同参考文献之间的行间距16磅，字体五号

**Comment by sastars9** [comment]
找到一种方法，但只能设置参考文献内部行间距，还是没法把不同条参考文献之间的间距设定好，请大佬指点
`\begingroup
\small % 设置字体大小
\setstretch{1.5} % 设置行间距
\printbibliography
\endgroup`

**Comment by note286** [comment, marked as answer]
请去阅读`biblatex-gb7714-2015`的文档。

**Reply by sastars9** [reply]
感谢，查到了。
